homeassistant/helpers/supervisor.py

```diff
@@ -0,0 +1,11 @@
+"""Supervisor helper."""
+
+import os
+
+from homeassistant.core import callback
+
+
+@callback
+def is_supervised() -> bool:
```

Comment:
So we have "supervised" as an installation type, this might be confusing.
`has_supervisor` maybe?

Reply:
Ah yea good point.